rest-server/core/src/main/scala/com/actionml/core/dal/CBGroupsDao.scala

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
This should be in the template package under the CB engine since it is Engine Specific. The Entire DAL/DAO/DAOImpl should be moved out of core. Once or if we find how to make these use Parameterized Types we may be able to move to core. But that is not required, I'd rather see if there is enough benefit from the extra work. We have no hard require to do yet another DB virtualization layer but it would make users job of creating Engines much easier if we could.